src/tests/index.test.ts

```diff
@@ -0,0 +1,70 @@
+import 'jest';
+
+/* eslint-disable @typescript-eslint/no-var-requires, global-require */
+
+const testPort = 3212;
+
+describe('Server', () => {
+  beforeAll(() => {
+    global.console = {
```

Comment:
Why override `console.log`? Couldn't we just set `LOG_LEVEL=emerg` for tests? Since there are no logs of that level we would hide the logs.

Reply:
Good catch, just made that change. I had no idea `console.log` listened to `LOG_LEVEL`, so TIL haha

For what it's worth, this is only here because of the uncontrollable errors thrown by the Bolt SDK when the API key is garbage.

Reply:
`console.log` doesn't, but we use `winston` around it and it does

Reply:
The error I was trying to suppress was Slack’s.... soooo maybe they use Winston?